### ISSUE TITLE
Cordage crafting line is a little out of whack

### ISSUE THREAD

**Issue by NyarlathotepZERO** [CleverRaven/Cataclysm-DDA#47398]
# Describe the bug

The line of cordage crafting is a little out of sync, especially the long cordage piece time being 5 minutes (the same as a short cordage piece)

This time being the same, offers somewhat of a crafting exploit for certain recipes, for example; when crafting a butchering rack, it makes sense to make the 2 long cordage pieces rather than 12 short cordage pieces. 

Also, on going over the recipes for the entire cordage crafting line, in the 'expected behaviour' section, I offer a suggestion for altering the 4 recipes to better line themselves as a more balanced progression of items. This is based largely on their length, then weighing that up against materials used and time.

This also adds a new option of using short cordage pieces throughout the entire line, just for completeness.

# Steps To Reproduce

Check the crafting time between long/short cordage pieces (they are the same)

# Expected behavior

short cordage piece (15cm) 5 minutes to make
	1x withered plant

long cordage piece (90cm) 30 minutes to make (6x short cordage piece @ 5 mins each)
	6x withered plant
	6x short cordage piece

short cordage rope (180cm) 1 hour to make (12x short cordage piece @ 5 mins each)
	12x withered plant
	12x short cordage piece
	2x long cordage piece

long cordage rope (900cm) 5 hours to make (60x short cordage piece @ 5 mins each)
	60x withered plant
	60x short cordage piece
	10x long cordage piece
	5x short cordage rope

# Screenshots

none

# Versions and configuration

Platform     : Windows_x64
Graphics     : Tiles
Build number : #11440
Build version: 0.E-9439-g1e8a075
Major version: 0.E

# Additional context

none

**Comment by Salty-Panda** [comment]
It makes sense to increase crafting times, making them similar to ropes/strings.

Did your check your recipe if resulted item is neither heavier nor bigger than total mass/volume of components used?

**Comment by NyarlathotepZERO** [comment]
.01
.04
.22
1.08
respectively

**Comment by NyarlathotepZERO** [comment]
Yeah, on doing the calculations the problem seems to be much bigger than this, and I will return to it when I can get something to look decent on paper.

In the interim, I suggest that if short/long cordage is not intended to hang carcasses, then remove all but 'long cordage rope' from the butchering rack recipe entirely. This would be a band-aid solution to solve the aforementioned exploit, this would also allow the description of the product used to fall more in line with this construction.
